checklog-odoo.cfg

```
@@ -0,0 +1,3 @@
+[checklog-odoo]
+ignore=
+	WARNING.*0 failed, 0 error\(s\).*
```

This would match `10 failed` (of course the tests would fail on those failures, but you know, details).

```suggestion
	WARNING.* 0 failed, 0 error\(s\).*
```